x-pack/filebeat/module/fortinet/firewall/ingest/traffic.json

```diff
@@ -0,0 +1,506 @@
+{
+  "description": "Pipeline for parsing fortinet firewall logs (traffic pipeline)",
+  "processors": [
+    {
+      "set": {
+        "field": "event.kind",
+        "value": "event"
+      }
+    },
+    {
+      "set": {
+        "field": "event.action",
+        "value": "{{fortinet.firewall.action}}",
+        "if": "ctx.fortinet?.firewall?.action != null"
+      }
+    },
+    {
+      "set": {
+        "field": "event.outcome",
+        "value": "success",
+        "if": "ctx.fortinet?.firewall?.action != null"
+      }
+    },
+    {
+      "append": {
+        "field": "event.category",
+        "value": "network"
+      }
+    },
+    {
+      "append": {
+        "field": "event.type",
+        "value": "connection"
+      }
+    },
+    {
+      "append": {
+        "field": "event.type",
+        "value": "start",
+        "if": "ctx.fortinet?.firewall?.action == 'start'"
+      }
+    },
+    {
+      "append": {
+        "field": "event.type",
+        "value": "end",
+        "if": "ctx.fortinet?.firewall?.action != 'start' && ctx.fortinet?.firewall?.action != null"
+      }
+    },
+    {
+      "append": {
+        "field": "event.type",
+        "value": "error",
+        "if": "['dns', 'ip-conn'].contains(ctx.fortinet?.firewall?.action)"
+      }
+    },
+    {
+      "append": {
+        "field": "event.type",
+        "value": "network",
```

Comment:
network isn't an event type, it is an event category though

Reply:
Typo, should have been protocol